content/20.treatments.md

```diff
@@ -22,7 +22,7 @@ In a healthy situation the lung respiratory epithelium together with alveolar ma
 It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines (TFN, IL-1𝛽) with the results of heavily damaging the endotelium of blood vessels and disrupting the balance between pro and anti-inflammatory response (Park et al. 2001).
 The holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) (Channappanavar and Perlman 2017), also caused by the new Corona virus . 
 
-Recently Chinese and Italian doctors have found that the the Tocilizumab (acterma, by Roche), a drug commonly used in the rheumatoid artitis, is very effective in the cure of Covid-19 patients. Specifically, Tocilizumab is a humanised monoclonal antibody that binds to the receptor of IL-6, de facto inhibiting the IL-6 immune cascade. Chinese doctors have started a trial enrolling 188 patients o which 14 with severe lung disease have shown clear sings of improvements, according to their results. Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection. Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April. The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyse its efficiency/safety (Biopharma-reporter.com).
+Recently Chinese and Italian doctors have found that the the Tocilizumab (acterma, by Roche), a drug commonly used in the rheumatoid arthritis, is very effective in the cure of Covid-19 patients. Specifically, Tocilizumab is a humanised monoclonal antibody that binds to the receptor of IL-6, de facto inhibiting the IL-6 immune cascade. Chinese doctors have started a trial enrolling 188 patients o which 14 with severe lung disease have shown clear sings of improvements, according to their results. Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection. Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April. The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyse its efficiency/safety (Biopharma-reporter.com).
```

Comment:
I think the Tocilizumab trial is exciting, but it may be worth mentioning both the pros and potential risks of using an immunosuppressive agent in the treatment of an infection. Typically, an immunosuppressive drug would be avoided in a person with a known infection because shutting down the immune system might allow the infection to spread. For example, this paper demonstrates the higher risk of infections in tocilizumab users with rheumatoid arthritis: https://academic.oup.com/rheumatology/article/51/5/852/1804905  Other papers also show risk of re-activation of Hep B in rheumatoid arthritis patients on tocilizumab: https://onlinelibrary.wiley.com/doi/abs/10.1111/1756-185X.13010

In this case, however, the severe "cytokine storm" in response to the virus is deleterious in and of itself, hence the possible benefit of using an IL-6 blocker in the short-term to curtail the effects of the virus. As a result, there's a delicate balance between blocking a harmful immune cascade (potentially allowing other infections to take over) and preventing worsening of the viral infection. I think a brief discussion of the risk-benefit analysis (why the trial might work, and what its potential drawbacks are) may be warranted. Perhaps this might mean that the treatment is suitable only when patients are at greatest risk of fatality ("severe lung disease" patients used in the study), and the immediate effects of the cytokine storm on worsening lung disease must be stopped before it's too late. Of course, these last thoughts are just my speculations; we can't know until the trial results are out.

Reply:
Dear dianerafi, I integrated your suggestions starting at line 38. Please ket me know your feedback as soon as you get a chance.
Best regards

Reply:
Looks good to me. I'll update if anything else comes to mind. I found the safety indications for TCZ here https://www.actemra.com/safety.html in case it's of interest (i.e. that the drug is usually contraindicated in patients with infection). It would be interesting if there is any evidence for TCZ being used to fight an infection rather than an autoimmune response, though I haven't been able to find that so far.